resources/aws/elb.go

```diff
@@ -225,7 +225,7 @@ func (lb ELB) findExisting() (*elb.LoadBalancerDescription, error) {
 	descriptions := resp.LoadBalancerDescriptions
 
 	if len(descriptions) == 0 {
-		return nil, NamedResourceNotFoundError{Name: lb.Name}
+		return nil, NotFoundError{ResourceType: ELBType, Name: lb.Name}
```

Comment:
I think the idea here is to keep error messages consistent, which I think is good. That's also why `type resourceType string` is here.